tests/fixtures/debian/kafka-streams-examples/standalone-network.yml

```diff
@@ -0,0 +1,117 @@
+---
+version: '2'
+
+networks:
+  # The name of the network defined here is used in `tests/test_kafka_streams_examples.py`
+  # as part of the 'NetworkMode' host config for some tests when running docker commands.
+  acme:
+    driver: bridge
+
+services:
+
+  ###
+  ### Bridge mode
+  ###
+  ### In this file, we use the port convention for bridge mode:
+  ### - Host port      = 2<usual_port> (example: 29092 for a Kafka broker that runs at port 9092)
+  ### - Container port = 1<usual_port> (example: 19092 for a Kafka broker that runs at port 9092)
+
+  zookeeper-bridge:
+    image: confluentinc/cp-zookeeper:3.2.0
+    labels:
+    - io.confluent.docker.testing=true
+    networks:
+    - acme
+    ports:
+    - "22181:12181"
```

Comment:
FYI : I think this is only needed if want to access these ports from outside (I mean not from within a docker container). You can simplify port management by keeping the ports as default because each container is essentially an independent network entity on a bridged network.

But you don't have to change this. Also, we can reuse this for a "Docker for Mac" tutorial with minimal changes.